packages/apps/human-app/server/src/modules/oracle-discovery/spec/oracle-discovery.service.spec.ts

```diff
@@ -45,6 +47,12 @@ describe('OracleDiscoveryService', () => {
         }),
       ],
       providers: [
+        {
+          provide: KvStoreGateway,
+          useValue: {
+            getJobTypesByAddress: jest.fn().mockReturnValue(JOB_TYPES),
```

Comment:
you should not be mocking this entire function, you should mock `OperatorUtils.getReputationNetworkOperators` and `this.cacheManager.get<OracleDiscoveryResponse[]>`inside it

Reply:
Actually, `getJobTypesByAddress` only calls `await KVStoreUtils.get`, and since we are not testing `KVStoreUtils` in the `oracle-discovery` tests, we likely don't need to mock any functions inside `getJobTypesByAddress`, only `getJobTypesByAddress` itself.